### DISCUSSION TITLE
Unable to create multi-line subtitle

### DISCUSSION THREAD

**Discussion by 412Linux** [Lednerb/bilberry-hugo-theme#625] [Ideas]
My objective was to create a multi-line subtitle.  From the  research I performed I thought this was possible using the three single quote method.  The site renders without issue.  However, the subtitle is not broken up as expected.  

Hugo version: hugo v0.123.1-3f8434a62fc91a50f4e5bd43a4435d2d301844ff+extended linux/amd64 BuildDate=2024-02-21T08:17:45Z VendorInfo=snap:0.123.1 

My toml file configuration for the subtitle is this:

   subtitle = '''
   This is a multi-line
   subtitle for my blog.
   It can span multiple lines
   without any issues.
   '''

The end result is is this.  No line breaks. 
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/113999425/87c28874-32dd-400e-8cd0-caab0b8517d3)

The index file looks like this: 
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/113999425/2e2db6e8-a81e-4401-95b7-7de46f00d2e1)

If this functionality isn’t supported by the theme just let me know and close it.  Thanks in advance for your time.

**Comment by igor-baiborodine** [comment]
@412Linux, this feature is not supported. I'm converting this issue to Discussions/Ideas as a nice-to-have that might be implemented in the future.
